packages/jupyter-ai/jupyter_ai/actors/generate.py

```diff
@@ -210,9 +202,16 @@ class GenerateActor(BaseActor):
     def __init__(self, reply_queue: Queue, root_dir: str, log: Logger):
         super().__init__(log=log, reply_queue=reply_queue)
         self.root_dir = os.path.abspath(os.path.expanduser(root_dir))
-        self.llm = ChatOpenAINewProvider(model_id='gpt-3.5-turbo')
+        self.llm = None
+
+    def create_llm_chain(self, provider: BaseProvider, provider_params: Dict[str, str]):
```

Comment:
The type for `provider` should actually be `Type[BaseProvider]`, since it should be the class object itself, and not an instance of that class.